apps/browser/src/_locales/en/messages.json

```
@@ -1561,7 +1561,10 @@
   "typeIdentity": {
     "message": "Identity"
   },
-  "newItemHeader":{
+  "typeSSHKey": {
+    "message": "SSH Key"
```

Follow pattern of other strings, Lowercase words after the first. See "Secure note" and "Password history"

```suggestion
    "message": "SSH key"
```